community/contributing/docs_writing_guidelines.rst

```diff
@@ -247,6 +248,31 @@ sentence, and keep it short:
 
     The **AtlasTexture's** used region.
 
+Use the Oxford comma to enumerate anything
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From the Oxford dictionary:
+
+    The 'Oxford comma' is an optional comma before the word 'and' at the end of a list:
+    *We sell books, videos, and magazines.*
+
+    [...] Not all writers and publishers use it, but it can clarify the meaning of a sentence when the items in a list are not single words:
+    *These items are available in black and white, red and yellow, and blue and green.*
+
+**Don't** leave the last element of a list without a comma:
+
+::
+
+    Create a KinematicBody2D node, a CollisionShape2D node and a sprite node.
+
+**Do** add a comma before `and` or `or`, for the last
+element of a list with more than two elements.
+
+::
+
+    Create a KinematicBody2D node, a CollisionShape2D node and, a sprite node.
```

Comment:
The comma should be before the "and", not after it, unless I am missing something.